homeassistant/components/forked_daapd/media_player.py

```diff
@@ -427,10 +433,12 @@ async def async_turn_on(self) -> None:
             futures = []
             for output in self._last_outputs:
                 futures.append(
-                    self._api.change_output(
-                        output["id"],
-                        selected=output["selected"],
-                        volume=output["volume"],
+                    asyncio.create_task(
```

Comment:
This change is just due to the deprecation of directly passing coroutines to asyncio.wait()